dashboard/views.py

```diff
@@ -422,100 +417,97 @@ def deduce_lyrics(track_names, track_artists, track_ids):
 
 
 def lyrics_vectorize(lyrics_vibes):
-    lyrics_constrain = [
```

Comment:
So it means that with HF endpoint, we no longer have lyrics constraints?

Reply:
Its moved to find_closest_emotion

Reply:
Got it